.idea/deploymentTargetDropDown.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="deploymentTargetDropDown">
+    <targetSelectedWithDropDown>
```

Comment:
Move to gitignore file .idea/deploymentTargetDropDown.xml (Read about gitignore)